modules/adding-a-data-connection-to-your-data-science-project.adoc

```diff
@@ -33,6 +33,16 @@ The *Add data connection* dialog opens.
 . In the *Access key* field, enter the access key ID for your S3-compatible object storage provider.
 . In the *Secret key* field, enter the secret access key for the S3-compatible object storage account you specified.
 . In the *Endpoint* field, enter the endpoint of your S3-compatible object storage bucket.
++
+NOTE: Use the appropriate endpoint format for your object storage type. Improper formatting might cause connection errors or restrict access to storage resources. 
+ifdef::upstream[]
+For more information on how to format object storage endpoints, see 
+link:{odhdocshome}/managing-resources/#overview-of-object-storage-endpoints[Overview of object strange endpoints]
+endif::[]
+ifndef::upstream[]
+For more information on how to format object storage endpoints, see 
+link:{rhoaidocshome}/managing-resources/#overview-of-object-storage-endpoints[Overview of object strange endpoints]
```

Comment:
Missing period at end. Is there a hard return after "see"?

Reply:
Fixed all three, nice spot!